blog/2023-08-10-evaluating-open-source-llms/index.md

```diff
@@ -0,0 +1,148 @@
+
+The reason is that the datasets used for these benchmarks are limited and do not cover all of the possible inputs an LLM could receive. Truth is, models developed by OpenAI, Cohere, and Anthropic are way larger in size (100B + parameters). On top of that, they are trained on way more data compared to their open-source counterparts. 
+
+So, the key takeaway is to use benchmarks as a starting point for evaluating LLMs, but not rely on them entirely. You want to focus on your specific LLM use case and understand the requirements for your project. If you don't have sensitive data or need full control over your LLM, using ChatGPT could allow you to build quickly while having top-tier performance and no infra setup.
+
+On the other hand, if privacy and security are absolutely required, then you can host your own open-source LLM. Along with using the benchmarks, you'll have to experiment with a handful of LLMs yourself to get a better idea of how each performs on live data. 
+
+
+## Reinforcement Learning
+
+Open-source models don't have the same size as ChatGPT or GPT-4, but even the larger 70B parameter models can have issues related to bias and fairness. When GPT-3 was first released it has [racial biases, gender biases](https://aclanthology.org/2021.nuse-1.5/), and [religious biases](https://hai.stanford.edu/news/rooting-out-anti-muslim-bias-popular-language-model-gpt-3). The origin of these biases was the data these models were trained on.
+
+It's nearly impossible to remove all biases from a dataset, but at the same time, we don't want the models to pick up on them. How do we fix this problem?
+
+This is where [*Reinforcement Learning with Human Feedback(RLHF)*](https://wandb.ai/ayush-thakur/RLHF/reports/Understanding-Reinforcement-Learning-from-Human-Feedback-RLHF-Part-1--VmlldzoyODk5MTIx) comes in. With RLHF, a human ranks the outputs of an LLM from best to worst. Each time the human ranks the outputs, they are essentially training a reinforcement model. This reinforcement model is then used to "reward" the model when it generates an output that is "good" and penalizes the model when the output is "bad".
+
+Using RLHF, the hidden biases a model has picked up from the training dataset can be removed, hence improving the model's accuracy.
+
+Pros:
+- Increased efficiency: Using RLHF, the feedback that is provided helps guide the LLM toward the optimal solution. With only a few examples, a fine-tuned model with RLHF can easily outperform the baseline model on certain tasks.
+- Better Performance: The feedback that a human provides also impacts the quality of the output generated by an LLM. By showing more examples of the desired outcomes, the LLM improves the generated output to match what is expected.
```

Comment:
these explanations don't make it clear what the difference between "efficiency" and "performance" is. They read like nearly identical bullet points. Do you actually mean "accuracy" and "quality"?